code/service/AlertService.php

```diff
@@ -29,14 +31,15 @@ public function getAlertsConfigContent($project) {
 	/**
 	 * Given a project and environment, sync the alerts configuration with the alerts gateway.
 	 *
-	 * @param DNProject $project
 	 * @param DNEnvironment $environment
+	 * @param string $sha
 	 * @param DeploynautLogFile $log
+	 * @param DNProject $project
 	 *
-	 * @return boolean
+	 * @return bool
 	 */
-	public function sync($project, $environment, $log) {
-		$content = $this->getAlertsConfigContent($project);
+	public function sync($environment, $sha, $log, $project) {
```

Comment:
The reason I changed order of this is to make it consistent with what the extension is recieving